### ISSUE TITLE
Wrong domain blocked

### ISSUE THREAD

**Issue by Quorum75** [privacy-protection-tools/anti-AD#675]
This domain clinique-veterinaire-desmettre-fath.fr is a vet and should not be blocked

**Comment by Ra2-IFV** [comment]
True, It's not an advertisement website
Fix it plz!

**Comment by Ra2-IFV** [comment]
Well, it seems that the domain wasn't on the block list.
You sure?

**Comment by Quorum75** [comment]
Yes, see my NextDNS log below. When I add the website to my whitelist it works. 

![image](https://user-images.githubusercontent.com/49731083/181658557-f25f7a46-eb5c-43c9-8748-1f78f4458703.png)

**Comment by avatartw** [comment]
www.clinique-veterinaire-desmettre-fath.fr

Blocked by CNAME
Original response: CNAME: web.jimdo.com

Rule(s): 
||jimdo.com^
